packages/vite-plugin-cloudflare/e2e/README.md

```diff
@@ -19,15 +21,15 @@ These tests use a mock npm registry where the built plugin has been published.
 
 The registry is booted up and loaded with the local build of the plugin and its local dependencies in the global-setup.ts file that runs once at the start of the e2e test run, and the server is killed and its caches removed at the end of the test run.
 
-The Vite `test` function is an extended with additional helpers to setup clean copies of fixtures outside of the monorepo so that they can be isolated from any other dependencies in the project.
+The Vite `test` function is extended with additional helpers to setup clean copies of fixtures outside of the monorepo so that they can be isolated from any other dependencies in the project.
 
 The simplest test looks like:
 
 ```ts
 test("can serve a Worker request", async ({ expect, seed, viteDev }) => {
 	const projectPath = await seed("basic", "pnpm");
 
-	const proc = await viteDev(projectPath);
+	const proc = await viteDev("npm", "dev", projectPath);
```

Comment:
```suggestion
	const proc = await runLongLived("npm", "dev", projectPath);
```

(the same change needs to be applied to line 29 above 👆 )

Reply:
95f496a469fbd4510bfa3470079b43635c606d40